webpack.config.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 const path = require('path');
 const webpack = require('webpack');
+const assert = require("assert/");
```

Comment:
that trailing `/` and the use of double quotes instead of single is bothering me, lol